supervisor/supervisor.go

```diff
@@ -141,24 +141,24 @@ func (sv *Supervisor) getHyperPod(container string, spec *specs.Spec) (hp *Hyper
 		return nil, fmt.Errorf("The container %s is already existing", container)
 	}
 	for _, ns := range spec.Linux.Namespaces {
-		if ns.Path != "" {
-			if strings.Contains(ns.Path, "/") {
-				return nil, fmt.Errorf("Runv doesn't support path to namespace file, it supports containers name as shared namespaces only")
+		if ns.Type == "network" && len(ns.Path) > 0 {
```

Comment:
I think we have to. 
Imagine this "--net container:a --pid container:b --ipc container:c", we can't make a logical decision on which POD to join.

POD is concept from K8S and K8S will always join namespaces of same container, I think limiting it to network namespace will work fine.